### PR TITLE
XIVY-13821 Add news about notifications

### PR DESCRIPTION
**Pull request by weissreto**

### CODE REVIEW

File: src/app/pages/news/11.3/04-notificationApi.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ## Business Notification API {#notificationAPI}
 
-... notifications
+Just introduced in 11.2, we now extended the Notifications feature even more.
```
Comment (ivy-rew) [inline src/app/pages/news/11.3/04-notificationApi.md:3]: This introduction is hard to read. Trying to simplify it. Feel free to keep it as is.

> The Notification feature introduced with 11.2, received further extensions, to address issues reported by our precious early bird user base.

File: src/app/pages/news/11.3/04-notificationApi.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+
+* Keep the users of a process up to date by using the new business notification API and event.
+* Notification messages have been streamlined, improved, and look better than ever.
+* All the information needed to fulfill your daily work is only a click away with the new link in notification messages to the details pages of the corresponding case or task. 
+* Customize the task notification message to the needs of your process by configuring the used notification template for a task.
```
Comment (ivy-rew) [inline src/app/pages/news/11.3/04-notificationApi.md:8]: simpler to read?

> You can now customize the notification message of a task on its template.

File: src/app/pages/news/11.3/04-notificationApi.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 ## Business Notification API {#notificationAPI}
 
-... notifications
+Just introduced in 11.2, we now extended the Notifications feature even more.
+
+* Keep the users of a process up to date by using the new business notification API and event.
+* Notification messages have been streamlined, improved, and look better than ever.
+* All the information needed to fulfill your daily work is only a click away with the new link in notification messages to the details pages of the corresponding case or task.
```
Comment (ivy-rew) [inline src/app/pages/news/11.3/04-notificationApi.md:7]: easier?
> ... only a click away: we have put better links to continue your work on task and case notifications.

File: src/app/pages/news/11.3/04-notificationApi.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
 ## Business Notification API {#notificationAPI}
 
-... notifications
+Just introduced in 11.2, we now extended the Notifications feature even more.
+
+* Keep the users of a process up to date by using the new business notification API and event.
```
Comment (ivy-rew) [inline src/app/pages/news/11.3/04-notificationApi.md:5]: easier?
> The business notification API allows to distribute your own notifications to users of your application.

File: src/app/pages/news/11.3/04-notificationApi.md (hunk 2 above)
Comment (weissreto) [reply src/app/pages/news/11.3/04-notificationApi.md:8]: Trying to avoid using always "You can now ..."

**Comment by weissreto** [comment]
Since the text seems to be to complicated, I have tried to simplify it.

**Review by ivy-rew — APPROVED** [review]

**Comment by ivy-rew** [comment]
much easier to read; at least with my personal bias. thanks :smile:

**Pull request merged**